jsk_vil_ros/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
```

Comment:
can we merge this package to `jsk_perception`?

Reply:
This node is irregular in terms of setting up an API server, and at the same time, there are features in Vision and Language that are hard to classify as perceptions, such as Image Generation, so I thought it would be better to separate them. 
So, what is the best practice for `jsk_recognition` ? Adding these node files and launch files to `jsk_perception` and write docs about these nodes not working without API server ?

Reply:
i think vision and language is perception.
you can also put server and api program if you write documentation propery.

Reply:
@k-okada do you have any suggestions?

Reply:
Once the policy has been decided, the names of messages and packages are fixed accordingly. Some people want to use this node and I want to let them use it.

Reply:
I talked with @k-okada , and this PR would be merged into jsk_perception